src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -34,7 +34,6 @@ The .NET Foundation licenses this file to you under the MIT license.
 
   <!-- Set up the defaults for the compatibility mode -->
   <PropertyGroup>
-    <IlcGenerateStackTraceData Condition="$(IlcGenerateStackTraceData) == ''">true</IlcGenerateStackTraceData>
     <IlcScanReflection Condition="$(IlcScanReflection) == ''">true</IlcScanReflection>
   </PropertyGroup>
 
```

Comment:
DynamicCodeSupport in the list below is no longer used.

Reply:
> DynamicCodeSupport in the list below is no longer used.

This sets a property read by the SDK, no?

Reply:
Ah ok, I thought that it is left over from `FEATURE_DYNAMIC_CODE`.